workorders.lic

```diff
@@ -683,9 +683,11 @@ class WorkOrders
 
   def request_work_order(recipes, npc_rooms, npc, npc_last_name, discipline, logbook, diff)
     match_names = recipes.map { |x| x['name'] }
+    $i = 0
+    $num = 500
 
     stow_hands
-    loop do
+    while $i <= $num do
```

Comment:
You can do `500.times do` which would remove the need for the loop variables

Reply:
In that case, you would also need a `break`. I'm a fan of `500.times do`.